bar/processedData/data.ts

```diff
@@ -0,0 +1,68 @@
+    {age_group: '25-29', sex: 'male', indicator: '2', mean: 0.41, high: 0.42, low: 0.40},
+    {age_group: '30-34', sex: 'male', indicator: '2', mean: 0.35, high: 0.37, low: 0.33},
+    {age_group: '35-39', sex: 'male', indicator: '2', mean: 0.46, high: 0.48, low: 0.42},
+    {age_group: '40-44', sex: 'male', indicator: '2', mean: 0.32, high: 0.37, low: 0.25},
+    {age_group: '45-49', sex: 'male', indicator: '2', mean: 0.26, high: 0.29, low: 0.25},
+    {age_group: '50-54', sex: 'male', indicator: '2', mean: 0.15, high: 0.18, low: 0.13},
+    {age_group: '55-59', sex: 'male', indicator: '2', mean: 0.14, high: 0.15, low: 0.11},
+
+
+    {age_group: '0-4', sex: 'female', indicator: '3', mean: 0.20, high:0.22, low: 0.18},
+    {age_group: '5-9', sex: 'female', indicator: '3', mean: 0.10, high:0.14, low: 0.06},
+    {age_group: '10-14', sex: 'female', indicator: '3', mean: 0.12, high: 0.12, low: 0.08},
+    {age_group: '15-19', sex: 'female', indicator: '3', mean: 0.09, high: 0.11, low: 0.08},
+    {age_group: '20-24', sex: 'female', indicator: '3', mean: 0.08, high: 0.11, low: 0.05},
+    {age_group: '25-29', sex: 'female', indicator: '3', mean: 0.20, high: 0.22, low: 0.17},
+    {age_group: '30-34', sex: 'female', indicator: '3', mean: 0.16, high: 0.19, low: 0.11},
+    {age_group: '35-39', sex: 'female', indicator: '3', mean: 0.19, high: 0.24, low: 0.14},
+    {age_group: '40-44', sex: 'female', indicator: '3', mean: 0.21, high: 0.28, low: 0.13},
+    {age_group: '45-49', sex: 'female', indicator: '3', mean: 0.1, high: 0.13, low: 0.08},
+    {age_group: '50-54', sex: 'female', indicator: '3', mean: 0.08, high: 0.03, low: 0.10},
+    {age_group: '55-59', sex: 'female', indicator: '3', mean: 0.06, high: 0.11, low: 0.02},
+
+    {age_group: '0-4', sex: 'male', indicator: '3', mean: 0.25, high:0.30, low: 0.21},
+    {age_group: '5-9', sex: 'male', indicator: '3', mean: 0.15, high:0.2, low: 0.13},
+    {age_group: '10-14', sex: 'male', indicator: '3', mean: 0.1, high: 0.13, low: 0.05},
+    {age_group: '15-19', sex: 'male', indicator: '3', mean: 0.14, high: 0.04, low: 0.15},
+    {age_group: '20-24', sex: 'male', indicator: '3', mean: 0.2, high: 0.21, low: 0.19},
+    {age_group: '25-29', sex: 'male', indicator: '3', mean: 0.11, high: 0.12, low: 0.07},
+    {age_group: '30-34', sex: 'male', indicator: '3', mean: 0.15, high: 0.17, low: 0.13},
+    {age_group: '35-39', sex: 'male', indicator: '3', mean: 0.19, high: 0.23, low: 0.12},
+    {age_group: '40-44', sex: 'male', indicator: '3', mean: 0.22, high: 0.24, low: 0.2},
+    {age_group: '45-49', sex: 'male', indicator: '3', mean: 0.28, high: 0.29, low: 0.25},
+    {age_group: '50-54', sex: 'male', indicator: '3', mean: 0.25, high: 0.28, low: 0.13},
+    {age_group: '55-59', sex: 'male', indicator: '3', mean: 0.11, high: 0.15, low: 0.09}
+];
+
+//TO COME FROM METADATA / USER SELECTIONS
+export const chartMetadata = {
```

Comment:
Can we split this up into the bit that's returned from the API and the bit that's selected by the user, to come up with a spec for the API? Then the other bit is metadata about what the filters themselves are...

Reply:
> I prefer chartjs both visually and codewise, and it seems much quicker to render also - a clear winner! Want to take that one forward and flesh out the prototype to derive the full API spec?

Oh good, me too!

Reply:
Yep, going to put together some sample data and types next, and plug it into this.